openwisp_notifications/handlers.py

```diff
@@ -29,21 +29,23 @@ def notify_handler(verb, **kwargs):
 
     target_org = getattr(kwargs.get('target', None), 'organization_id', None)
 
-    where = Q()
+    where = Q(is_superuser=True)
+    where_group = Q()
     if target_org:
-        where = Q(is_staff=True) & Q(openwisp_users_organization=target_org)
+        where = where | (Q(is_staff=True) & Q(openwisp_users_organization=target_org))
+        where_group = Q(openwisp_users_organization=target_org)
+    where_group = where_group & Q(notificationuser__receive=True)
     where = where & Q(notificationuser__receive=True)
 
     if recipient:
         # Check if recipient is User, Group or QuerySet
         if isinstance(recipient, Group):
-            recipients = recipient.user_set.filter(where)
+            recipients = recipient.user_set.filter(where_group)
```

Comment:
My thought while implementing this was that if `target_org` is absent, then that will be some general notifications, so it might not be a problem.

Reply:
If i get you well, general notification means everyone in the system. hmmmmm. i doubt this will be demanded now. the most general case i can think of is all members of an organization. Also, when you look at most `openwisp` objects, they all must belong to an organization. `device` must have an organization, `template` must have an organization, even when you create a `metric` on `openwisp-monitoring` it must contain an object which is either directly or in-directly connected to an organization. so i think the most general case we can have for now is the organization level.

I think this is a great feature but don't think now is the right time to implement it.